.release-please-manifest.json

```diff
@@ -8,6 +8,7 @@
   "packages/snaefell-ui": "1.1.0",
   "packages/supplementary-contracts": "1.0.0",
   "packages/taiko-client": "2.0.0",
+  "packages/taiko-client-rs": "0.1.0",
```

Comment:
Maybe we should sync `taiko-client` and `taiko-client-rs` major versions to avoid confusion? @davidtaikocha